providers/apache/kafka/src/airflow/providers/apache/kafka/operators/consume.py

```diff
@@ -150,7 +148,7 @@ def execute(self, context) -> Any:
                 **self.apply_function_kwargs,
             )
 
-        messages_left = self.max_messages
+        messages_left = self.max_messages or True
```

Comment:
Have to use this because we still have the logic below.
https://github.com/apache/airflow/blob/d517c637f800d0034d9aecdac19a7521757c33e6/providers/apache/kafka/src/airflow/providers/apache/kafka/operators/consume.py#L158